code/modules/surgery/bone_mending.dm

```diff
@@ -0,0 +1,142 @@
+			"<span class='notice'>[user] successfully repairs the fracture in [target]'s [parse_zone(target_zone)]!</span>")
+		log_combat(user, target, "repaired a hairline fracture in", addition="INTENT: [uppertext(user.a_intent)]")
+		qdel(surgery.operated_wound)
+	else
+		to_chat(user, "<span class='warning'>[target] has no hairline fracture there!</span>")
+	return ..()
+
+/datum/surgery_step/repair_bone_hairline/failure(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, var/fail_prob = 0)
+	..()
+	if(istype(tool, /obj/item/stack))
+		var/obj/item/stack/used_stack = tool
+		used_stack.use(1)
+
+
+
+///// Reset Compound Fracture (Crticial)
+/datum/surgery_step/reset_compound_fracture
+	name = "reset bone"
+	implements = list(/obj/item/bonesetter = 100, /obj/item/stack/sticky_tape/surgical = 60, /obj/item/stack/sticky_tape/super = 40, /obj/item/stack/sticky_tape = 20)
+	time = 40
+	experience_given = MEDICAL_SKILL_MEDIUM
+
+/datum/surgery_step/reset_compound_fracture/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(surgery.operated_wound)
+		display_results(user, target, "<span class='notice'>You begin to reset the bone in [target]'s [parse_zone(user.zone_selected)]...</span>",
+			"<span class='notice'>[user] begins to reset the bone in [target]'s [parse_zone(user.zone_selected)] with [tool].</span>",
+			"<span class='notice'>[user] begins to reset the bone in [target]'s [parse_zone(user.zone_selected)].</span>")
+	else
+		user.visible_message("<span class='notice'>[user] looks for [target]'s [parse_zone(user.zone_selected)].</span>", "<span class='notice'>You look for [target]'s [parse_zone(user.zone_selected)]...</span>")
+
+/datum/surgery_step/reset_compound_fracture/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results = FALSE)
+	if(surgery.operated_wound)
+		if(istype(tool, /obj/item/stack))
+			var/obj/item/stack/used_stack = tool
+			used_stack.use(1)
+		display_results(user, target, "<span class='notice'>You successfully reset the bone in [target]'s [parse_zone(target_zone)].</span>",
+			"<span class='notice'>[user] successfully resets the bone in [target]'s [parse_zone(target_zone)] with [tool]!</span>",
+			"<span class='notice'>[user] successfully resets the bone in [target]'s [parse_zone(target_zone)]!</span>")
+		log_combat(user, target, "reset a compound fracture in", addition="INTENT: [uppertext(user.a_intent)]")
+	else
+		to_chat(user, "<span class='warning'>[target] has no compound fracture there!</span>")
+	return ..()
+
+/datum/surgery_step/reset_compound_fracture/failure(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, var/fail_prob = 0)
+	..()
+	if(istype(tool, /obj/item/stack))
+		var/obj/item/stack/used_stack = tool
+		used_stack.use(1)
+
+
+///// Repair Compound Fracture (Crticial)
```

Comment:
```suggestion
///// Repair Compound Fracture (Critical)
```

Reply:
wait, actually, a lot of these comments are copy+pasted

make sure they all accurately describe the surgery steps they're above